README.md

```
@@ -2023,50 +2021,38 @@ pipeline:
 1. **Batching** 100 messages *or* 100 ms – whichever comes first.
 2. **Sanitising** Illegal chars in the key become “\_”.
-3. **Headers** All Benthos metadata (except `kafka_*`) plus `bridged_by` are forwarded as Kafka headers.
-4. **Topic check** If `umh.messages` is missing the plugin creates it (1 partition, `compact,delete`).
+3. **Kafka/Redpanda** Each message will be stored in the Kafka topic `umh.messages` with the Kafka key of the umh_topic
+4. **Headers** All Benthos metadata (except `kafka_*`) plus `bridged_by` are forwarded as Kafka headers.
+5. **Topic check** If `umh.messages` is missing the plugin creates it (1 partition, `compact,delete`).
 
 #### Troubleshooting / FAQs
 
-* **“topic is not set or is empty”** – your pipeline never wrote `msg.meta.topic`.
+* **“topic is not set or is empty”** – your pipeline never wrote `msg.meta.umh_topic`.
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)